python/lsst/cp/pipe/cpCombine.py

```diff
@@ -464,7 +484,7 @@ def _subBBoxIter(bbox, subregionSize):
                                        (bbox, subregionSize, colShift, rowShift))
                 yield subBBox
 
-    def combine(self, target, expHandleList, expScaleList, stats):
+    def combine(self, target, expHandleList, expScaleList, stats, maskForNoData="BAD"):
```

Comment:
Does `maskForNoData` get used elsewhere?  Was this a change from prior to adding `self.config.noGoodPixelsMask`?

Reply:
Good catch.  This was from before I realized I needed to set it in the `stats` object.